templates/patterns/alert/alert.ui_patterns.yml

```diff
@@ -1,47 +1,42 @@
 alert:
-  label: Alert
-  description: Provide contextual feedback messages for typical user actions with the handful of available and flexible alert messages. https://v5.getbootstrap.com/docs/5.0/components/alerts/
+  label: 'Alert'
+  description: 'Provide contextual feedback messages for typical user actions with the handful of available and flexible alert messages. https://v5.getbootstrap.com/docs/5.0/components/alerts/'
   settings:
     style:
```

Comment:
Why we don't use variants for this? I think it fits more the scope of variants.

Reply:
Variant is a reserved word and doesn't work if there are no variants (patterns with different behaviours) in the pattern.